devnet/ops.json

```diff
@@ -0,0 +1,5 @@
+{
+"project_id" : "dummy",
+"client_email" : "dummy",
+"private_key" : "dummy"
+}
```

Comment:
What is this used for?

Reply:
used by nix check . will be removed later . there is TODO